web-server/v0.4/src/pages/Dashboard/Controllers.js

```diff
@@ -44,13 +46,9 @@ export default class Controllers extends Component {
 
     dispatch({
       type: 'global/fetchDatastoreConfig',
-    })
-      .then(() => {
-        this.fetchMonthIndices();
-      })
-      .catch(err => {
-        console.log(err);
-      });
+    }).then(() => {
+      this.fetchMonthIndices();
+    });
```

Comment:
Why are we dropping the `.catch()` clause which will log the error to the console?  Are we handling this some place else now?

Reply:
I didn't realize this was dropped and will check this back in. Eventually, we'll want to have a global error reducer that captures networking errors given the status of the request ("request", "success", "error") and logs them with Redux.